### PR TITLE
v1.18 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+ - Added a setting to the system console to change the minimum length of hashtags.
+ - Added a reply option to long press context menu.
+
+### Bug Fixes
+ - Fixed an issue where blank spaces broke markdown tables.
+ - Fixed an issue where deactivated users appeared on "Add Members" modal but not on the search results.
+ - Fixed an issue on Android where extra text in the search box appeared after using the autocomplete drop-down.
+ - Fixed an issue with multiple text entries when typing with Shift+Letter on Android.
+ - Fixed an issue where push notifications badges did not always clear when read on another device.
+ - Fixed an issue on iOS where opening a single or group notification did not take the user into the channel where the notification came from.
```
Comment (enahum) [inline CHANGELOG.md]: Both platforms IOs and Android

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+ - Fixed an issue where timezone did not automatically update on Android when travelling to another timezone.
+ - Fixed an issue where the user mention autocomplete drop-down was case sensitive.
+ - Fixed an issue where system admininistrators were able to see the full long press menu when long pressing a system message.
+ - Fixed an issue on iOS where users were not able to unflag posts from "Flagged Posts" when opened from a read-only channel.
```
Comment (enahum) [inline CHANGELOG.md]: Both platforms IOs and Android

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+ - Fixed an issue on iOS where users were unable to create channel names of 2 byte characters.
```
Comment (enahum) [inline CHANGELOG.md]: Both platforms IOs and Android

**Comment by amyblais** [comment]
@enahum @esethna Can you help review v1.18 changelog by tomorrow?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+ - Fixed an issue where push notifications badges did not always clear when read on another device.
+ - Fixed an issue where opening a single or group notification did not take the user into the channel where the notification came from.
+ - Fixed an issue where timezone did not automatically update on Android when travelling to another timezone.
+ - Fixed an issue where the user mention autocomplete drop-down was case sensitive.
+ - Fixed an issue where system admininistrators were able to see the full long press menu when long pressing a system message.
+ - Fixed an issue where users were not able to unflag posts from "Flagged Posts" when opened from a read-only channel.
+ - Fixed an issue where users were unable to create channel names of 2 byte characters.
+ 
+### Known Issues
+ - Content for ephemeral messages is not displayed on Mattermost Mobile Apps.
```
Comment (enahum) [inline CHANGELOG.md]: @amyblais we have a PR for this if we want to have it for 1.18 #2710

**Comment by amyblais** [comment]
@enahum Can you help merge this PR at some point today?

I don't know if Eric is going to review this (I've also tried to ping him).

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,42 @@
...
 
+## 1.18.0 Release
+- Release Date: April 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ - ``Bot`` tags were added for bot accounts feature in server v5.10 and mobile v1.18, meaning that mobile v1.17 and earlier don't support the tags.
```
Comment (esethna) [inline CHANGELOG.md]: @amyblais why is this necessary to put under compatibility? This seems like a new feature that should just go under improvements?

Comment (amyblais) [reply CHANGELOG.md]: It's a release note that v1.17 and earlier don't support bot tags, so people might think it's a bug.
